### PR TITLE
Use Kibana byte formatter with ECS bytes fields

### PR DESCRIPTION
**Pull request by andrewkroh**
Add a Kibana field formatter to the fields that contain bytes so that they are represented in human readable format (e.g. 10 KB, 10GB).

### CODE REVIEW

File: testing/environments/Makefile [hunk 1]
```diff
@@ -11,8 +11,8 @@ stop:
 
 
 up:
-	${BASE_COMMAND} build
-	${BASE_COMMAND} up
+	${BASE_COMMAND} build elasticsearch kibana
```
[the remark this answers is not in the document]
Comment (andrewkroh) [reply testing/environments/Makefile]: Yeah, thanks for catching this. I have undone this change.

[the remark this answers is not in the document]
Comment (ruflin) [reply testing/environments/Makefile]: Probably not pushed yet? Will also need an other rebase.

[the remark this answers is not in the document]
Comment (andrewkroh) [reply testing/environments/Makefile]: Pushed now. Was about to push then I realized it also needed a rebase then I got interrupted by a flight.

**Comment by andrewkroh** [comment]
An issue has be opened for ECS: https://github.com/elastic/ecs/issues/305

**Pull request merged**